link.go

```diff
@@ -19,30 +19,34 @@ var LinkCommand = cli.Command{
 	Usage: "Symlink packages to their dvcsimport repos, for local development.",
 	Description: `gx-go link eases local development by symlinking actual workspace repositories on demand.
 
-Example workflow:
+Links the specified dependency package (by name or hash) used by the
+parent package (in the CWD or specified by --parent). The link is done
```

Comment:
I'd prefer to maintain the parent-child relationship concept, because everything is a package here  *but*, depending on where you're looking from (in what part of the code you're in), that package is either a dependency (and that distinction is important because a dependency in a `package.json` doesn't have the same information as the actual package) or the package that's importing that dependency. The term "current" doesn't seem to carry that relationship meaning, is there another term you'd prefer?

Reply:
When I'm using GX, I'm usually working *on* a gx package. The "current" package is the package I'm currently working on.

For example:

> `gx-go link` replaces the target gx package (either by name or hash) with a symlink to the appropriate dvcs repo in your GOPATH. To make this "symlinked" repo usable as a gx package, `gx-go link` rewrites the target package's dvcs imports using the target package's `package.json`. Unfortunately, this can cause build errors in packages *depending* on this package if these dependent packages specify alternative, conflicting dependency versions. We can work around this by asking `gx` to rewrite the target package using dependencies from the current package (the package you're trying to build) first, falling back on the target package's `package.json` file. We can do this by passing the `--deps=mine` flag.

That *is* a bit wordy and you're right, "current" may not be quite right.

Reply:
Let's leave it at "the current dependent package". Nice text (I like it *wordy* :), I'll add it to the usage doc.